### PR TITLE
Update FSM scaffold tool to add scaffolded skill to `packages.json`

### PR DESCRIPTION
**Pull request by angrybayblade**
## Proposed changes

Updates the FSM scaffold tool to update the `packages.json` after scaffolding the skill

## Fixes

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

### CODE REVIEW

File: autonomy/cli/scaffold_fsm.py [hunk 1]
```diff
@@ -99,4 +99,8 @@ def fsm(ctx: Context, registry: str, skill_name: str, spec: str) -> None:
     scaffold_item(ctx, SKILL, skill_name)
     ctx.cwd = preserve_cwd
 
-    ScaffoldABCISkill(ctx, skill_name, Path(spec)).do_scaffolding()
+    scaffold_fsm = ScaffoldABCISkill(ctx, skill_name, Path(spec))
+    scaffold_fsm.do_scaffolding()
+
+    if ctx.config[TO_LOCAL_REGISTRY_FLAG]:
+        scaffold_fsm.add_skill_to_packages()
```
Comment (angrybayblade) [inline autonomy/cli/scaffold_fsm.py:106]: @jmoreira-valory This will update the `packages.json` after scaffolding the skill

Comment (jmoreira-valory) [reply autonomy/cli/scaffold_fsm.py:106]: will this add to 'dev' or 'third_party' ?

Comment (angrybayblade) [reply autonomy/cli/scaffold_fsm.py:106]: dev

Comment (dvilelaf) [reply autonomy/cli/scaffold_fsm.py:106]: Looking at the function definition on line 361 it seems to dev packages, which is correct.

**Review by dvilelaf — APPROVED** [review]

**Review by jmoreira-valory — APPROVED** [review]

**Pull request merged**